third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-view-trace-run-expected.txt

```diff
@@ -72,7 +72,6 @@ uses-text-compression
 uses-webp-images
 
 View Trace Button Text: "View Original Trace"
-View Trace Button Title: "The performance metrics above are simulated and won't match the timings found in this trace. Disable simulated throttling in "Lighthouse Settings" if you want the timings to match."
-
-Showing view: timeline
+PROMISE FAILURE: TypeError: UI.Tooltip.getContent is not a function
```

Comment:
I think this webtest was fixed recently. Either way, we shouldn't commit the error into our expectations.